src/context/Symbols.h

```diff
@@ -118,12 +119,36 @@ class SymbolTable final {
         }
     }
 
+    struct VarUser {
+        int64_t id;
+        bool    inLoop;
+    };
```

Comment:
Why not place this usage info into Variable structure ?

Reply:
Good catch. Modified.